src/main/java/com/novoda/notils/logger/simple/Log.java

```diff
@@ -206,6 +209,13 @@ public static void setShowLogs(boolean showLogs) {
      * @return {@code true} if logs are active, {@code false} if deactivated
      */
     public static boolean shouldShowLogs() {
-        return SHOW_LOGS;
+        if (INITIALISED) {
+            return SHOW_LOGS;
+        } else {
+            throw new DeveloperError("#rekt - To use simple logger you need to have called setShowLogs(boolean). "
```

Comment:
true but want to keep this logger as simple as possible with least overhead. (as close to the android log as possible). Think an instance might make a user question that, also if we hid the instance from the user (keeping the same api) then I feel it might become bloated - don't want people not using the because of performance